src/Native/Unix/System.Native/pal_networking.c

```diff
@@ -1635,6 +1635,10 @@ static bool TryGetPlatformSocketOption(int32_t socketOptionName, int32_t socketO
                     *optName = IPV6_MULTICAST_IF;
                     return true;
 
+                case SocketOptionName_SO_IP_MULTICAST_TTL:
```

Comment:
Just out of curiosity, why is it that `IP_MULTICAST_TTL` and `IP_MULTICAST_IF` are supported for both `SOL_IP` and `SOL_IPV6`, but IP_MULTICAST_LOOP is only supported on `SOL_IP`?

Reply:
I did not look at IP_MULTICAST_LOOP nor did I do any comprehensive testing.  (there is open issue to do that)